qgis-app/styles/views.py

```diff
@@ -43,6 +47,92 @@ def check_styles_access(user, style):
     return False
 
 
+def send_mail_wrapper(subject,
+                      message,
+                      mail_from,
+                      recipients,
+                      fail_silently=True):
+    """
+    Wrapping send_email function to send email only when not DEBUG.
+    """
+
+    if settings.DEBUG:
+        logging.debug("Mail not sent (DEBUG=True)")
+    else:
+        send_mail(subject,
+                  message,
+                  mail_from,
+                  recipients,
+                  fail_silently)
+
+
+def style_notify(style, created=True):
+    """
+    Email notification when a new style created.
+    """
+    recipients = [u.email for u in User.objects.filter(
+        is_staff=True, email__isnull=False).exclude(email='')]
+
+    if created:
+        style_status = "created"
+    else:
+        style_status = "updated"
+
+    if recipients:
+        domain = Site.objects.get_current().domain
+        mail_from = settings.DEFAULT_FROM_EMAIL
+
+        send_mail_wrapper(
+            _('A new style has been %s by %s.') % (style_status,
+                                                   style.creator),
+            _('\r\nStyle name is: %s\r\nStyle description is: %s\r\n'
+              'Link: http://%s%s\r\n') % (style.name, style.description,
+                                          domain, style.get_absolute_url()),
+            mail_from,
+            recipients,
+            fail_silently=True)
+        logging.debug('Sending email notification for %s style, '
+                      'recipients:  %s' % (style.name, recipients))
+    else:
+        logging.warning('No recipients found for %s style notification'
+                        % style.name)
+
+
+def style_approval_notify(style, creator):
+    """
+    Email notification system for approval styles
+    """
+
+    recipients = [creator.email]
+
+    if settings.QGIS_DEV_MAILING_LIST_ADDRESS:
```

Comment:
Again here, we should only notify the style manager group members and the individual original submitter of the style.